web/src/stream_settings_ui.ts

```diff
@@ -1178,4 +1178,11 @@ export function initialize(): void {
         $(".right").removeClass("show");
         $("#channels_overlay_container .two-pane-settings-header").removeClass("slide-left");
     });
+
+    $("#channels_overlay_container").on("click", "#preview-stream-button", () => {
+        const stream_id = Number.parseInt($(".stream_settings_header").attr("data-stream-id")!, 10);
+        if (!Number.isNaN(stream_id)) {
```

Comment:
Why this condition is needed? We can always expect `data-stream-id` to be a number.